app/src/androidTest/java/org/mozilla/fenix/ui/SitePermissionsTest.kt

```diff
@@ -196,12 +190,12 @@ class SitePermissionsTest {
     }
 
     @Test
+    @Ignore("Firebase - No camera and microphone on AVD")
     fun rememberBlockCameraPermissionChoiceTest() {
```

Comment:
@AaronMT I'm currently working on permissions tests for Focus and I also looked for a method to check if the device has a camera available. So far this looks promising (I only tested locally on an AVD with no camera/one camera):
```
private val cameraManager = getTargetContext.getSystemService(Context.CAMERA_SERVICE) as CameraManager
in test: 
Assume.assumeTrue(cameraManager.cameraIdList.isNotEmpty())
```

Reply:
safe to disable until we have more info https://github.com/mozilla-mobile/mobile-test-eng/issues/622